storage/migrations/02_runtimes.up.sql

```diff
@@ -117,8 +117,8 @@ CREATE TABLE chain.runtime_events
   -- `evm_log_params` fields below.
   body JSONB NOT NULL,
   evm_log_name TEXT,
-  -- The event signature, if it exists, will be the first topic.
-  evm_log_signature TEXT GENERATED ALWAYS AS (body->'topics'->>0) STORED,
+  -- The event signature, present only for known event types.
```

Comment:
Maybe we can get rid of this comment now and instead incorporate it into line 117.
And/or somehow visually separate the `evm_*` fields and not that they are only populated for events whose `type='evm.log'` AND their ABI is known, which happens roughly when the emitting contract is verified.

Reply:
Done